### PR TITLE
Docker configs

### PR DESCRIPTION
**Pull request by ismukhin**
@valentina-kustikova @a-sidorova @FenixFly , посмотрите, пожалуйста, конфиги. Я их писал по примеру с документации `OpenVINO`. Но я немного в сомнениях: при сборке, например, `tf2` после отработки `accuracy_check` появляется значение `accuracy` равный примерно 30%, при этом в `MXNet` и `ONNXRuntime` около 60%. Это нормально? Так и должно быть?

### CODE REVIEW

**Comment by valentina-kustikova** [comment]
> @valentina-kustikova @a-sidorova @FenixFly , посмотрите, пожалуйста, конфиги. Я их писал по примеру с документации `OpenVINO`. Но я немного в сомнениях: при сборке, например, `tf2` после отработки `accuracy_check` появляется значение `accuracy` равный примерно 30%, при этом в `MXNet` и `ONNXRuntime` около 60%. Это нормально? Так и должно быть?

@ismukhin, точность должна быть примерно одинаковой, отличия максимум во втором знаке после запятой. Посмотрела конфиги, принципиальных ошибок я не нашла. @a-sidorova, посмотрите, пожалуйста, если время позволяет.

**Comment by a-sidorova** [comment]
> > @valentina-kustikova @a-sidorova @FenixFly , посмотрите, пожалуйста, конфиги. Я их писал по примеру с документации `OpenVINO`. Но я немного в сомнениях: при сборке, например, `tf2` после отработки `accuracy_check` появляется значение `accuracy` равный примерно 30%, при этом в `MXNet` и `ONNXRuntime` около 60%. Это нормально? Так и должно быть?
> 
> @ismukhin, точность должна быть примерно одинаковой, отличия максимум во втором знаке после запятой. Посмотрела конфиги, принципиальных ошибок я не нашла. @a-sidorova, посмотрите, пожалуйста, если время позволяет.

Да, там везде [одинаковая](https://github.com/openvinotoolkit/open_model_zoo/tree/2022.3.0/tools/accuracy_checker/sample) должна быть - `75.02%`.
Если у вас есть время, я могу на неделе самостоятельно глянуть подробнее на предложенные конфигурации и потестировать (сегодня-завтра). Возможно какие-то параметры в конфигурационном файле пропущены или наоборот лишние для конкретного фреймворка. Поскольку это `SampleNet` (без понятия, что за модель), наверно придется просто подобрать параметры.

**Comment by valentina-kustikova** [comment]
> > > @valentina-kustikova @a-sidorova @FenixFly , посмотрите, пожалуйста, конфиги. Я их писал по примеру с документации `OpenVINO`. Но я немного в сомнениях: при сборке, например, `tf2` после отработки `accuracy_check` появляется значение `accuracy` равный примерно 30%, при этом в `MXNet` и `ONNXRuntime` около 60%. Это нормально? Так и должно быть?
> > 
> > 
> > @ismukhin, точность должна быть примерно одинаковой, отличия максимум во втором знаке после запятой. Посмотрела конфиги, принципиальных ошибок я не нашла. @a-sidorova, посмотрите, пожалуйста, если время позволяет.
> 
> Да, там везде [одинаковая](https://github.com/openvinotoolkit/open_model_zoo/tree/2022.3.0/tools/accuracy_checker/sample) должна быть - `75.02%`. Если у вас есть время, я могу на неделе самостоятельно глянуть подробнее на предложенные конфигурации и потестировать (сегодня-завтра). Возможно какие-то параметры в конфигурационном файле пропущены или наоборот лишние для конкретного фреймворка. Поскольку это `SampleNet` (без понятия, что за модель), наверно придется просто подобрать параметры.

Да, посмотрите, пожалуйста, у нас нет срочности. Спасибо!

**Comment by a-sidorova** [comment]
Я сравнила `ONNX` и `OV` модели. Архитектуры одинаковые, а вот веса разные. Возник вопрос, а как так получилось после конверсии в IR. Ради интереса я воспользовалась `ONNX` фронтендом в `OV` (на вход прям подала модель в формате `.onnx`) - и получила метрику качества абсолютно такую же, как в `ONNX`. То же самое получила, когда я сама сконвертировала с помощью `Model Optimizer` модель в `IR`. Тогда я обратилась с вопросом к инженерам OMZ и выяснила следующее:

Очень давным-давно на заре `AccuracyChecker` кто-то написал простой перцептрон и обучил с помощью `Caffe`. Потом понадобились модели в формате других библиотек и началась конверсия из одного формата в другой. Тогда `OpenVINO` не поддерживало ни `ONNX`, ни `PyTorch`, ни `TF2`, а получить эти модели нужно было. Пришлось заново создавать модель и ее тренировать. Ну и соответственно для каких-то библиотек одно значение метрики, для других - другое. Тут просто дело в том, что на самом деле в моделях просто лежат разные веса 🙂 

Так мне кажется, что можно оставить как есть. Поскольку это просто какая-то `sample` модель, обученная исключительно для демонстрации работы `AccuracyChecker`, а не какая-нибудь открытая модель.

**Comment by valentina-kustikova** [comment]
> Я сравнила `ONNX` и `OV` модели. Архитектуры одинаковые, а вот веса разные. Возник вопрос, а как так получилось после конверсии в IR. Ради интереса я воспользовалась `ONNX` фронтендом в `OV` (на вход прям подала модель в формате `.onnx`) - и получила метрику качества абсолютно такую же, как в `ONNX`. То же самое получила, когда я сама сконвертировала с помощью `Model Optimizer` модель в `IR`. Тогда я обратилась с вопросом к инженерам OMZ и выяснила следующее:
> 
> Очень давным-давно на заре `AccuracyChecker` кто-то написал простой перцептрон и обучил с помощью `Caffe`. Потом понадобились модели в формате других библиотек и началась конверсия из одного формата в другой. Тогда `OpenVINO` не поддерживало ни `ONNX`, ни `PyTorch`, ни `TF2`, а получить эти модели нужно было. Пришлось заново создавать модель и ее тренировать. Ну и соответственно для каких-то библиотек одно значение метрики, для других - другое. Тут просто дело в том, что на самом деле в моделях просто лежат разные веса 🙂
> 
> Так мне кажется, что можно оставить как есть. Поскольку это просто какая-то `sample` модель, обученная исключительно для демонстрации работы `AccuracyChecker`, а не какая-нибудь открытая модель.

@a-sidorova, спасибо за помощь. В таком случае действительно имеет смысл оставить все как есть, нас интересует работоспособность, но при этом мы теперь понимаем, почему расхождение в точностях.

**Review by valentina-kustikova — APPROVED** [review]

File: docker/TensorFlow/Dockerfile [hunk 1]
```diff
@@ -7,19 +7,24 @@ RUN pip3 install --upgrade pip && \
...
-
 WORKDIR /tmp/
 COPY models.lst models.lst
 RUN python3 ./open_model_zoo/tools/model_tools/downloader.py --list models.lst
 RUN rm models.lst
 
 WORKDIR /tmp/open_model_zoo/tools/accuracy_checker
-RUN python3 setup.py install_core
-RUN accuracy_check -c /tmp/dl-benchmark/docker/TensorFlow/config.yml -m data/test_models -s sample
+COPY config_tf2.yml config_tf2.yml
```
Comment (a-sidorova) [inline docker/TensorFlow/Dockerfile:16]: Раньше (и сейчас для `caffe`) мы не копировали конфиг файл, а использовали путь из репозитория. Как я понимаю сейчас мы копируем просто потому, что в мастере у нас пока нет еще этих файлов, чтобы мы их смогли достать в контейнере. Я бы предложила после мержа тогда создать PR, который убирал это копирование и удаление, и вместо этого использовал бы путь относительно контейнера. Что думаете?

**Review by a-sidorova — APPROVED** [review]

File: docker/TensorFlow/Dockerfile (hunk 1 above)
Comment (ismukhin) [reply docker/TensorFlow/Dockerfile:16]: > Раньше (и сейчас для `caffe`) мы не копировали конфиг файл, а использовали путь из репозитория. Как я понимаю сейчас мы копируем просто потому, что в мастере у нас пока нет еще этих файлов, чтобы мы их смогли достать в контейнере. Я бы предложила после мержа тогда создать PR, который убирал это копирование и удаление, и вместо этого использовал бы путь относительно контейнера. Что думаете?

Да, я так и хотел сделать, поэтому создам отдельный PR и подправлю этот момент. Спасибо!

Comment (valentina-kustikova) [reply docker/TensorFlow/Dockerfile:16]: Тогда я заливаю эти изменения.
